### DISCUSSION TITLE
Use Latest Version of Wagmi with Etherjs

### DISCUSSION THREAD

**Discussion by meetkosiso** [wevm/wagmi#2438] [Q&A]
I'm using the latest version of Wagmi that supports viem, but our code base is written with etherjs, want to confirm if there are a possibility that I can use the latest version of wagmi with ethersjs.

**Comment by tmm** [comment]
You are welcome to continue using ethers along with viem, but wagmi requires viem moving forward.

**Reply by layinka** [reply]
Ok, how do i get a Signer and provider from the public/wallet client that i can pass to ethers?
